client/coral-admin/src/components/ReplyBadge.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {Badge} from 'coral-ui';
+import t from 'coral-framework/services/i18n';
+
+const ReplyBadge = ({hasParent}) => hasParent ?
```

Comment:
Imo the logic for checking `hasParent` should not lie within `ReplyBadge`. It should not receive a `hasParent` property because what does that mean for the `ReplyBadge`? What is the parent of a reply badge? That's logic that should reside at the parent component who renders the `ReplyBadge`.